index.html

```diff
@@ -962,7 +962,8 @@ <h3>
               [=this=].{{HID/[[devices]]}}:
                 <ol>
                   <li>If the user has allowed the site to access |device| as
-                  the result of a previous call to {{HID/requestDevice()}},
+                  the result of a previous call to {{HID/requestDevice()}} and
+                  |device|.{{HIDDevice/[[state]]}} is not `"forgotten"`,
```

Comment:
{{HID/requestDevice()}} should reset {{HIDDevice/[[state]]}} to "closed" for any returned HIDDevice instances in case the devices were previously forgotten.

Reply:
You're absolutely right. Good catch!